Made changes at lines 103 and 105 of event3.html

kiddjsh opened the pull request:
The following changes were made to the event3.html file:

@ 103 Added space between Location: Manhattan
@ 103 Capitalized New Jersey
@ 105 Added space between efforts. Where
@ 105 Capitalized the letter p in Please
@ 105 Corrected the word wild to WildGuard

rahulkhandait-sde commented:
Conflicting files
event2.html
event3.html

@kiddjsh Please resolve the conflicts in the mentioned files. Ensure you pull all the latest changes before pushing your feature/bug fixes to the main branch.

kiddjsh commented:
closing to resubmit

kiddjsh commented:
@rahulkhandait-sde I have updated the pull request and removed file event2.html which appeared to be causing the conflict. Could you verify and merge as quickly as possible as I have other updates to submit for review. Thanks!
